### ISSUE TITLE
Keyboard interactions of videos in featured content module appear to focus on invisible elements

### ISSUE THREAD

**Issue by stephanieosan** [cfpb/design-system#1124]
## Bug

When tabbing through a blog page that has a video, the navigation through the video focuses on several invisible elements and the preview image jumps up on at least one of them:

![Video-Bug](https://user-images.githubusercontent.com/6433178/102245933-24685580-3ecc-11eb-82bc-9ef1b310f4d8.gif)

See https://www.consumerfinance.gov/about-us/blog/guide-coronavirus-mortgage-relief-options/ or https://www.consumerfinance.gov/about-us/blog/protect-yourself-financially-from-impact-of-coronavirus/ as examples.

### More context
Is this a problem with the blog page template, or with videos generally?

Looks to be when we use our custom video component. Like this page with the youtube default skin doesn't have the issue https://www.consumerfinance.gov/consumer-tools/retirement/ whereas this page's video does https://www.consumerfinance.gov/consumer-tools/managing-someone-elses-money/

**Comment by chosak** [comment]
Debugging the tab stops on [this page](https://www.consumerfinance.gov/consumer-tools/managing-someone-elses-money/), the extra elements seem to be:

- the hidden close button (`button.a-btn.o-video-player_close-btn`)
- the video iframe (`iframe#widget2.o-video-player_iframe`)

Adding `tabindex="1"` should hopefully fix these. We'll have to add/remove that to the close button depending on if the video is playing or not.

**Comment by chosak** [comment]
PR opened at https://github.com/cfpb/consumerfinance.gov/pull/6208.
